docs/_install-and-build/Installing-Geth.md

```diff
@@ -3,310 +3,411 @@ title: Installing Geth
-automatically use the data of your old node and sync the latest blocks that were mined
-since you shutdown the old software.
+## Package managers
 
-## Install from a package manager
+### MacOS via Homebrew
 
-### Install on macOS via Homebrew
+The easiest way to install go-ethereum is to use the Geth Homebrew tap. The first step is to check that Homebrew is installed. The following command should return a version number.
 
-The easiest way to install go-ethereum is to use our Homebrew tap. If you don't have
-Homebrew, [install it first][brew].
+<br>
+```shell
+brew -v
+```
+<br>
 
-Run the following commands to add the tap and install `geth`:
+If a version number is returned, then Homebrew is installed. If not, Homebrew can be installed by following the instructions [here][brew]. With Homebrew installed, the following commands add the Geth tap and install Geth:
 
+<br>
 ```shell
 brew tap ethereum/ethereum
 brew install ethereum
 ```
+<br>
 
-You can install the master branch using the `--devel` parameter:
 
+The previous command installs the latest stable release. Developers that wish to install the most up-to-date version can install the Geth repository's master branch by adding the `--devel` parameter to the install command:
+
+<br>
 ```shell
 brew install ethereum --devel
 ```
+<br>
+
+These commands install the core Geth software and the following developer tools: `clef`, `devp2p`, `abigen`, `bootnode`, `evm`, `rlpdump` and `puppeth`. The binaries for each of these tools are saved in `/usr/local/bin/`. The full list of command line options can be viewed in the terminal by running `geth --help`.
+
+Updating an existing Geth installation to the latest version can be achieved by stopping the node and running the following commands:
+
+<br>
+```shell
+brew update 
+brew upgrade 
+brew reinstall ethereum
+```
+<br>
 
-The `abigen`, `bootnode`, `checkpoint-admin`, `clef`, `devp2p`, `ethkey`, `evm`, `faucet`, `geth`, `p2psim`, `puppeth`, `rlpdump`, and `wnode` commands are then available on your system in `/usr/local/bin/`.
+When the node is started again, Geth will automatically use all the data from the previous version and sync the blocks that were missed while the node was offline.
 
-Find the different options and commands available with `geth --help`.
 
-### Install on Ubuntu via PPAs
+### Ubuntu via PPAs
 
-The easiest way to install go-ethereum on Ubuntu-based distributions is with the built-in
-launchpad PPAs (Personal Package Archives). We provide a single PPA repository that
-contains both our stable and development releases for Ubuntu versions `trusty`, `xenial`,
-`zesty` and `artful`.
+The easiest way to install Geth on Ubuntu-based distributions is with the built-in launchpad PPAs (Personal Package Archives). A single PPA repository is provided, containing stable and development releases for Ubuntu versions `trusty`, `xenial`, `zesty` and `artful`.
```

Comment:
The ubuntu versions seem to be outdated. From the [PPA](https://launchpad.net/~ethereum/+archive/ubuntu/ethereum) page:

xenial, trusty, impish, focal, bionic

Reply:
thanks for the catch, updating now.